lib/spack/spack/modules.py

```diff
@@ -527,8 +537,13 @@ class TclModule(EnvModule):
     path = canonicalize_path(
         _roots.get(name, join_path(spack.share_path, 'modules')))
 
+    def autoload_warner(self):
+        if self.verbose_autoload():
+            return 'puts stderr "Autoloading {module_file}"\n'
```

Comment:
I disagree about this and the next one.  I think that the knowledge of how deeply indented the string will be when it gets used is local to the place where it gets used.  If something changed about the way that the hunk of code contained in 'autoload_format'  that changed the indentation, your approach would require changes in both locations (or, in my case a bug that I'd have to bang my head against for a while before finding it... ;().